decidim-comments/app/mailers/decidim/comments/comment_notification_mailer.rb

```diff
@@ -1,44 +0,0 @@
-# frozen_string_literal: true
-
-module Decidim
-  module Comments
-    # A custom mailer for sending notifications to users when
-    # a comment is created.
-    class CommentNotificationMailer < Decidim::ApplicationMailer
```

Comment:
I'm not sure about this, maybe we can make the event class know about which mailer use so that we can further customize how the emails look

Reply:
I think we don't need this right now because I just passed some extra information to the event and rendered different messages.
We can include this option in a future PR.